### DISCUSSION TITLE
how to use the types?

### DISCUSSION THREAD

**Discussion by sqllyw** [vuejs/petite-vue#210] [Q&A] [answered]
Hi,
npm run build, it generates a dist/types, where to put this folder so that typescript can find it? thanks

**Comment by sqllyw** [comment, marked as answer]
found a way, create a file petite-vue.es.d.ts with following content:
````
export { createApp } from './types/app';
export { nextTick } from './types/scheduler';
export { reactive } from '@vue/reactivity';

````
put it in the same folder as petite-vue.es, the folder content will be like following:

![image](https://github.com/vuejs/petite-vue/assets/29809419/528c851a-dd67-4ea7-9813-0b2c94902006)

**Comment by sebimoe** [comment]
I have the same issue and find modifying the package dist files is not ideal for me.

I am adding `"types": "./dist/types/index.d.ts",` into exports of package.json and `export * from './directives';` into `dist/types/index.d.ts` - if someone would like to make a PR with it would be much appreciated!
